src/screens/PetRegistration/index.tsx

```diff
@@ -367,11 +342,12 @@ const PetRegistration = () => {
 	})
 
 	const handleSendData = async () => {
```

Comment:
Consegue colocar a navegação pra alguma outra pagina no caso de sucesso ? To bem perdido com a navegação